lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -138,6 +138,11 @@ def debug_message
         else
           sorted_logs = container_logs.sort_by { |_, log_lines| log_lines.length }
           sorted_logs.each do |identifier, log_lines|
+            if log_lines.empty?
+              helpful_info << "  - Logs from container '#{identifier}': #{DEBUG_RESOURCE_NOT_FOUND_MESSAGE}"
```

Comment:
I think that telling the user there are no logs is information that is useful for debugging.

From an ideal perspective, i like the unix "no news is good news" though. I could go either way here - I guess it depends how often this might come up in the logs.